utility/test/gov_test.go

```diff
@@ -13,6 +13,9 @@ import (
 	"google.golang.org/protobuf/types/known/wrapperspb"
 )
 
+// TODO(andrew): Remove the use of `require.True` and `require.False` in all cases.
+// TODO(andrew): Use require.Equal and avoid the use of formatted strings. Ditto elsewhere.
+// TODO(andrew): Use reflection to iterate over all the params and test them.
```

Comment:
Suggestion:  [Google's go-cmp](https://github.com/google/go-cmp) provides useful routines to do comparison of expected and actual structs, with outputs that are easy to read, especially when the number of fields is large.

Reply:
Nice suggestion - probably better than using reflection.

Just for context, most of these TODO(andrew) came from Olshansk and were just moved from a different file

Reply:
I'll add the suggestion in the comment